notebook/static/notebook/js/notebook.js

```diff
@@ -728,8 +736,18 @@ define(function (require) {
         var selectedIndex = Math.min(Math.max(this.get_selected_index() + offset, 0), this.ncells()-1);
         this.select(selectedIndex);
         this.get_selected_cell().marked = true;
+        this.update_marked_status()
     };
 
+    Notebook.prototype.update_marked_status = function(){
```

Comment:
This function gets changed substantially in #683 anyway... why don't I remove it from here for now and we can figure out what to do with it in #683.